packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.ts

```diff
@@ -36,6 +38,67 @@ function bytesOrTextToString(obj: IRgBytesOrText): string {
         obj.text;
 }
 
+/**
+ * Attempts to build an validate an absolute path from a given pattern an a root folder
+ * If a first pass is not successful a second pass will consider finding it by removing
+ * a glob suffix '/**', if a valid path is successfully built the suffix can be appended
+ * back to the result string i.e. depending on the keepSuffix flag.
+ *
+ * @param root
+ * @param pattern
+ * @param keepSuffix
+ */
+function findPathInSuffixedPattern(root: string, pattern: string, keepSuffix: boolean): string | undefined {
+    let suffix = undefined;
+    let searchPath = findPathInPattern(root, pattern);
+
+    if (searchPath) {
+        return searchPath;
+    }
+
+    // Attempt to find a path in provided pattern but without a glob suffix
```

Comment:
nitpick: for consistency

```suggestion
    // Attempt to find a path in provided pattern but without a glob suffix.
```

Reply:
Done.